src/cluster/src/lib.rs

```diff
@@ -21,9 +21,11 @@
 mod helm;
 mod install;
 mod error;
+mod releases;
 
 pub use install::ClusterInstaller;
 pub use install::ClusterInstallerBuilder;
 pub use error::ClusterError;
+pub use releases::list_releases;
```

Comment:
I think it might make sense to make the `HelmClient` publicly available rather than making one-off functions like this. What if we make all of the `HelmClient` methods into `pub(crate)` visibility and only expose `HelmClient::new()` and `HelmClient::versions(&self, name: &str) -> Vec<Chart>` as the fully `pub` functions? Then we can add

```rust
pub use helm::HelmClient;
```

into `lib.rs`